action.yml

```diff
@@ -34,6 +34,17 @@ inputs:
       https://docs.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token
     default: ${{ github.token }}
     required: false
+  excludeTargetBranch:
+    description: >
+      When set to `true` excludes commits from the target branch. This is useful on merge commits of two
+      divergent branches. By default all of the commits in the target branch that are not in the source
+      branch will be linted again. Set this to `true` to avoid that.
+
+      TODO: This explanation can be made better when I better understand why `--firstParent` alone didn't resolve this.
+      To do that I need to understand what `git log <revision range>` is doing.
+    default: "false"
+    required: false
```

Comment:
if we understand the root problem better, then perhaps this should default to `true` much like `firstParent`.